ecommerce/src/main/kotlin/io/hhplus/ecommerce/coupon/application/handler/CouponEventHandler.kt

```
@@ -109,4 +119,38 @@ class CouponEventHandler(
+                    "couponName" to payload.couponName,
+                    "requestedAt" to payload.requestedAt
+                )
+            )
+
+            logger.info(
+                "[CouponEventHandler] Kafka 발행 완료: couponId={}, userId={}",
+                payload.couponId, payload.userId
+            )
+
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)